grails-app/controllers/org/pih/warehouse/core/LocationController.groovy

```diff
@@ -83,6 +83,11 @@ class LocationController {
                 }
             }
 
+            Organization currentOrganization = locationInstance.organization
+            Organization pickedOrganization = Organization.get(params.organization?.id)
```

Comment:
Sorry for the nitpick but let's called this selectedOrganization. The term "picked" has a different meaning in a WMS  so I don't want to confuse anyone.